### PR TITLE
chore(test): migrated from jest to vitest

### PR DESCRIPTION
**Pull request by ghalibansari**

### CODE REVIEW

File: tests/adapter.node.spec.ts [hunk 1]
```diff
@@ -25,45 +43,59 @@ describe('axiosCacheAdapter tests (node)', () => {
...
-            };
-            (httpAdapter as jest.Mock).mockImplementationOnce(async () =>
-                Promise.resolve(responseWithHeader),
-            );
-            await cacheAdapterWithParseHeaders(config);
-            expect(cacheSetSpy).toHaveBeenCalled();
-        },
-    );
+
+    // it.each(['max-age', 's-maxage'])(
```
Comment (ghalibansari) [inline tests/adapter.node.spec.ts:77]: @Goldziher 
i have fixed all test and migrated to vite and vitest,
but im not able to fix this specific test it.each having some, plz me with this to solve it.

File: package.json [hunk 2]
```diff
@@ -26,44 +26,45 @@
...
-        "test:coverage": "jest --coverage",
-        "test:watch": "jest --watch",
+        "test": "vitest run",
+        "test:coverage": "vitest run --coverage",
+        "test:watch": "vitest",
         "prepare": "husky install"
     },
     "peerDependencies": {
-        "axios": ">=0.13.0"
+        "axios": ">=1.5.0"
```
Comment (Goldziher) [inline package.json:38]: this shouldn't be changed

Comment (ghalibansari) [reply package.json:38]: ok will change it to 0.27.2, below. version are depcrecated

Comment (ghalibansari) [reply package.json]: or should i change it back to 0.13.0

Comment (Goldziher) [reply package.json]: lowest non-deprecated

**Pull request merged**